File: ClientApp/src/app/core/models/doctor.model.ts
export interface Doctor {
  id: number;
  department: string;
  nationalId: string;
  fullName: string;
  gender: number;
  dateOfBirth: Date;
  phonenumber: string;
  speciality: string;
  email: string;
  workingStartDate: Date;
  workingEndDate: Date;
  createdBy: string;
  createdDate: Date;
  updatedBy: string;
  updatedDate: Date;
  isDeleted: boolean;
}

export interface iDoctorDetails {
  id: number;
  userId: string;
  address: string;
  dateOfBirth: string;
  departmentId: string;
  departmentName: string;
  email: string;
  fullName: string;
  avatarUrl: string;
  gender: string;
  nationalId: string;
  phoneNumber: string;
  speciality: string;
  workingEndDate: string;
  workingStartDate: string;
  createdDate: string;
  updatedDate: string;
}

File: WebAPI/WebAPI/Models/DoctorModel.cs
namespace WebAPI.Models
{
    public class DoctorModel
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string NationalId { get; set; }
        public string Gender { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string PhoneNumber { get; set; }
        public string Speciality { get; set; }
        public string WorkingStartDate { get; set; }
        public string WorkingEndDate { get; set;}
    }
}
